### DISCUSSION TITLE
Use pretrained torchscript file to continue training

### DISCUSSION THREAD

**Discussion by joetorsney** [acids-ircam/RAVE#214] [Q&A] [answered]
Hi there,

I'm interested in using the pre-trained RAVE model on the VCTK speech dataset [that is available as a torchscript](https://acids-ircam.github.io/rave_models_download).

I understand that you can train RAVE using a checkpoint file such as `rave train --ckpt /my-checkpoint.ckpt`. Is it possible to do something similar with a torchscript file? Or load the torchscript file in and make a checkpoint? 

Thanks in advance!

**Comment by joetorsney** [comment, marked as answer]
I'm pretty sure that this isn't possible unless you can travel back in time and get the version of the source code that was used to train the VCTK example. Also, you can only get the state_dict after loading in the torchscript and it seems RAVEs checkpoints have loads more stuff.

tldr it's not possible?
